app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -159,6 +159,10 @@ private void setButton() {
             findViewById(R.id.activity_description_stats_data_layout).setVisibility(View.GONE);
            }
         } else {
+            recButton.setVisibility(View.INVISIBLE);
+            recButton.setEnabled(false);
+            findViewById(R.id.activity_description_stats_layout).setVisibility(View.GONE);
```

Comment:
So thats in the case where you're not registered, right ?

Reply:
+ 1

Reply:
Exactly !